src/pages/ServiceDetails/ServiceInfo/IstioObjectDetails/DestinationRuleDetail.tsx

```diff
@@ -37,7 +37,7 @@ class DestinationRuleDetail extends React.Component<DestinationRuleProps> {
     let message = checks.map(check => check.message).join(',');
 
     if (!message.length) {
-      if (validation && !validation.valid) {
+      if (!validation.valid) {
```

Comment:
Is this really the same? Something like `!undefined.method` returns a `TypeError` on the console.

Reply:
Was wondering the same, but validation is never undefined up to this point, check line [30](https://github.com/jotak/swsui/blob/2d90dc92c79d38d7da1caeb3a52456c15f64863d/src/pages/ServiceDetails/ServiceInfo/IstioObjectDetails/DestinationRuleDetail.tsx#L30)

Reply:
yes and normally typescript is smart enough to report an error if we potentially access a field from undefined variable (unless it's `any`)